core/src/main/java/org/mskcc/cbio/portal/servlet/QueryBuilder.java

```diff
@@ -425,10 +477,11 @@ private void processData(CohortDetails cohortDetails,
         Boolean hasMethylation = false;
         Boolean hasCopyNo = false;
         Boolean hasSurvival = false;
-
+        
         // retrieve samples
-        Map<String, Set<String>> inputStudySampleMap = cohortDetails.getStudySampleMap();
-        if (inputStudySampleMap.keySet().size() == 1 && !cohortDetails.getIsVirtualStudy()) { // single study
+        Map<String, Set<String>> inputStudySampleMap = studySampleMap1;
```

Comment:
why do we create inputStudySampleMap?

Reply:
Thanks for pointing it! will clean it up